SPECS/sympa-6.2.spec.in

```diff
@@ -547,17 +562,26 @@ fi
 %license COPYING
 %dir %attr(-,sympa,sympa) %{_sysconfdir}/sympa/
 %config(noreplace) %attr(0640,sympa,sympa) %{_sysconfdir}/sympa/sympa.conf
-%config(missingok) %{_sysconfdir}/sympa/create_list_templates
```

Comment:
Having all the config files available make it easier to understand how to tune sympa behaviour, so I'd like to keep them. But as you outlined, these config files are actually not mandatory, so I'll go with %config(noreplace,missingok) which should allow for both. If file are removed or moved upstream, it would need to be acommodated for in the specfile anyway, so this shouldn't be an issue.